### ISSUE TITLE
Vehicle Tracker Stop Working?

### ISSUE THREAD

**Issue by srvoets** [bjhiltbrand/drone_mobile_home_assistant#22]
as stated, my device tracker for vehicle is reading unknown, as well as temp and a few other sensors.

lock/unlock/panic/start works.

I removed integration, and relaoded re added signed in, same thing.

**Comment by joshuaferrara** [comment]
ditto on my instance as well. i'm seeing this log message:
```
2022-08-06 23:59:18.915 ERROR (MainThread) [homeassistant.components.device_tracker] Error adding entities for domain device_tracker with platform drone_mobile
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 428, in async_add_entities
await asyncio.gather(*tasks)
File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 673, in _async_add_entity
await entity.add_to_platform_finish()
File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 775, in add_to_platform_finish
await self.async_added_to_hass()
File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 341, in async_added_to_hass
self._handle_coordinator_update, self.coordinator_context
AttributeError: 'CarTracker' object has no attribute 'coordinator_context'
```

**Comment by joshuaferrara** [comment]
Well, I haven't ever hacked on HomeAssistant code before - so I'm not sure if this is a proper fix or just a band-aid one - but adding this line (29) and restarting HomeAssistant got my GPS data flowing again:

<img width="353" alt="image" src="https://user-images.githubusercontent.com/248680/183341299-50c36a8b-7ab2-49ae-8d51-48954947e988.png">

**Comment by bjhiltbrand** [comment]
Sorry guys, I missed this fix when I fixed the switch and sensor classes earlier. I don't have the tracker stuff on my DroneMobile subscription anymore, so I never saw an issue in the logs or in the integration. Thanks for pointing it out. I simply applied the same fix that I did for the other classes here, for consistency. I also did a little refactoring and renaming, so switches may have changed names in your installs.
